modules/stochastic_tools/include/distributions/NormalDistribution.h

```diff
@@ -35,11 +35,11 @@ class NormalDistribution : public Distribution
 protected:
   ///@{
   /// Coefficients for the rational function used to approximate the quantile
-  const std::vector<Real> _a = {
-      -0.322232431088, -1.0, -0.342242088547, -0.0204231210245, -0.0000453642210148};
+  const std::vector<Real> _a = {{
+      -0.322232431088, -1.0, -0.342242088547, -0.0204231210245, -0.0000453642210148}};
 
-  const std::vector<Real> _b = {
-      0.099348462606, 0.588581570495, 0.531103462366, 0.10353775285, 0.0038560700634};
+  const std::vector<Real> _b = {{
```

Comment:
These really should be std::arrays - stack versus heap allocation which can help with cache hits.

Reply:
I guess I didn't read the description before adding the comment.

Reply:
I added this little fix before I realized that you had reverted the original PR, so this is an std::array but I am fixing the parallel stuff. This PR will get replaced in a short while.